lib/modules/apostrophe-doc-type-manager/lib/api.js

```diff
@@ -31,9 +32,6 @@ module.exports = function(self, options) {
       // So methods called later, or extending this method, see the change in piece
       doc[name] = $set[name];
     });
-    if (_.isEmpty($set)) {
```

Comment:
This has to come back so that if the doc has no fields that need deduplicating we don't wind up with an empty $set which is a fatal error in mongo.

Reply:
k